src/game/server/tf/bot/behavior/tf_bot_use_item.cpp

```diff
@@ -47,9 +47,12 @@ ActionResult< CTFBot >	CTFBotUseItem::Update( CTFBot *me, float interval )
 	{
 		if ( m_cooldownTimer.IsElapsed() )
```

Comment:
I know this is a bit of format nitpicking, but the if statements can be combined into one like so.
```suggestion
		if ( m_cooldownTimer.IsElapsed() && me->GetNextAttack() <= gpGlobals->curtime )
```